packages/gitgraph-react/src/Gitgraph.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { GitGraphOptions, Commit } from "gitgraph-core/lib/index";
```

Comment:
My biggest hack for now… I don't why, but it's not works only with `gitgraph-core`… If someone have an idea (normally the package.json is well configure, but I probably missed something)